native/machine-coding/bmi-calculator/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>BMI Calculator</title>
+  <link rel="stylesheet" type="text/css" href="styles.css">
+  <script src="../../helpers/header.js" type="module"></script>
+  <script src="script.js" defer></script>
+</head>
+<body>
+  <main>
+    <h1>BMI Calculator</h1>
+    <div class="calculator">
+      <label for="height">Height (in meters):</label>
+      <input type="text" id="height" placeholder="Enter your height" step="0.01">
+      <label for="weight">Weight (in kilograms):</label>
+      <input type="text" id="weight" placeholder="Enter your weight" step="0.01">
+      <button onclick="calculateBMI()">Calculate BMI</button>
```

Comment:
Instead of binding the function in html, please use `addEventListener` in the script.
You may also use the `form` element and use the `submit` event of the form in the `addEventListener`

Reply:
done